src/day-1-toy/lists.py

```diff
@@ -8,22 +8,25 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # [command here]
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
-# [command here]
+x += y
```

Comment:
No assignment (`=`) operators! Try looking for a list method that can do the same thing in the Python docs.

Reply:
Oh, gotcha.